README.md

```diff
@@ -1 +1,21 @@
-# microverse-project-hallo
\ No newline at end of file
+# Hello-Microverse-Project
+
+# Project Name
+Hello Microverse project
+
+# Built With
+HTML & CSS
+Linters
+
+# Getting Started
```

Comment:
[ ] kindly add instructions on setting up your project locally. (example: clone the repo by git clone "...", cd file .....)🤓

Reply:
I'm done adding instructions on setting up my project locally.